docs/03-tutorials/serverless/svls-06-sync-function-with-gitops.md

```diff
@@ -66,7 +64,7 @@ These sections will lead you through the whole installation, configuration, and
   kyma init function --runtime python38 --dir $PWD/{WORKSPACE_FOLDER}
   ```
 
-  > **TIP:** Python 3.8 is only one of the available runtimes. Read about all [supported runtimes and sample Functions to run on them](#details-runtimes).
+  >**TIP:** Python 3.8 is only one of the available runtimes. Read about all [supported runtimes and sample Functions to run on them](../../../05-technical-reference/svls-01-sample-functions.md).
```

Comment:
⚠️  Link to `svls-01-sample-functions.md `does not work. 
(However, it did work when you used it in svls-02-from-code-to-function.md )